SPECS/systemd/systemd.spec

```diff
@@ -1,62 +1,345 @@
-Distribution:   Mariner
-Group:          System Environment/Security
-URL:            https://www.freedesktop.org/wiki/Software/systemd/
-Source0:        https://github.com/%{name}/%{name}-stable/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        95-disable-systemd-oomd.preset
-BuildRequires:  audit-devel
-BuildRequires:  cryptsetup-devel
-BuildRequires:  docbook-dtd-xml
-BuildRequires:  docbook-style-xsl
-BuildRequires:  gettext
-BuildRequires:  glib-devel
-BuildRequires:  gperf
-BuildRequires:  intltool
-BuildRequires:  kbd
-BuildRequires:  kmod-devel
-BuildRequires:  libcap-devel
-BuildRequires:  libgcrypt-devel
-BuildRequires:  libselinux-devel
-BuildRequires:  libxslt
-BuildRequires:  lz4-devel
+
+# FIXME - hardcode to 'stable' for now as that's what we have in our blobstore
+%global stable 1
+# %%global stable %%(c="%%version"; [ "$c" = "${c#*.*}" ]; echo $?)
+
+# For a breakdown of the licensing, see README
+License:        LGPL-2.1-or-later AND MIT AND GPL-2.0-or-later
+Summary:        System and Service Manager
+
+%if %{undefined version_no_tilde}
+# This is defined in 'macros.rust-srpm' which we may not have installed.
+# However, we don't plan on using any RC releases, so we should be able to just define it directly to the version.
+%define version_no_tilde %version
+%endif
+
+# download tarballs with "spectool -g systemd.spec"
+%if %{defined commit}
+Source0:        https://github.com/systemd/systemd%{?stable:-stable}/archive/%{commit}/%{name}-%{shortcommit}.tar.gz
+%else
+%if 0%{?stable}
+Source0:        https://github.com/systemd/systemd-stable/archive/v%{version_no_tilde}/%{name}-%{version_no_tilde}.tar.gz
+%else
+Source0:        https://github.com/systemd/systemd/archive/v%{version_no_tilde}/%{name}-%{version_no_tilde}.tar.gz
+%endif
+%endif
+# This file must be available before %%prep.
+# It is generated during systemd build and can be found in build/src/core/.
+Source1:        triggers.systemd
+Source2:        split-files.py
+
+# Prevent accidental removal of the systemd package
+Source4:        yum-protect-systemd.conf
+
+Source6:        sysctl.conf.README
+Source7:        systemd-journal-remote.xml
+Source8:        systemd-journal-gatewayd.xml
+Source9:        20-yama-ptrace.conf
+Source10:       systemd-udev-trigger-no-reload.conf
+
+Source14:       10-oomd-defaults.conf
+Source15:       10-oomd-per-slice-defaults.conf
+Source16:       10-timeout-abort.conf
+Source17:       10-map-count.conf
+
+Source21:       macros.sysusers
+Source22:       sysusers.attr
+Source23:       sysusers.prov
+Source24:       sysusers.generate-pre.sh
+
+Source25:       98-default-mac-none.link
+
+%if 0
+GIT_DIR=../../src/systemd/.git git format-patch-ab --no-signature -M -N v235..v235-stable
+i=1; for j in 00*patch; do printf "Patch%04d:      %s\n" $i $j; i=$((i+1));done|xclip
+GIT_DIR=../../src/systemd/.git git diffab -M v233..master@{2017-06-15} -- hwdb/[67]* hwdb/parse_hwdb.py >hwdb.patch
+%endif
```

Comment:
If it's not used, let's remove it. Or please add a comment explaining why we need to keep it.
```suggestion
```

Reply:
reducing delta from our inherited spec file is far more important that removing this just for the sake of its removal without any actual technical change.